win32/sendmail.c

```diff
@@ -333,6 +333,31 @@ PHPAPI char *GetSMErrorText(int index)
 	}
 }
 
+/* strtok_r like, but recognizes quoted-strings */
+static char *find_address(char *list, char **state)
+{
+	zend_bool in_quotes = 0;
+	char *p = list;
+
+	if (list == NULL) {
+		if (*state == NULL) {
+			return NULL;
+		}
+		p = list = *state;
+	}
+	*state = NULL;
+	while ((p = strpbrk(p, ",\"")) != NULL) {
+		if (*p == '\"' && (p <= list ||*(p - 1) != '\\')) {
```

Comment:
I don't think this handles escaping correctly. This will not interpret `\"` as a quote, but `\\"` should be interpreted as a quote.

If you encounter a `\`, you should be skipping over the next character.

Reply:
Ah, right. Thank you!